src/ORM/FieldType/DBPrimaryKey.php

```diff
@@ -2,64 +2,59 @@
 
 namespace SilverStripe\ORM\FieldType;
 
+use SilverStripe\Forms\FormField;
 use SilverStripe\ORM\DataObject;
 use SilverStripe\ORM\DB;
+use SilverStripe\View\ViewableData;
 
 /**
  * A special type Int field used for primary keys.
  */
 class DBPrimaryKey extends DBInt
 {
-    /**
-     * @var DataObject
-     */
-    protected $object;
+    protected ?DataObject $object;
```

Comment:
Just like with `DBForeignKey`, this is explicitly intended to represent a relation to a `DataObject` record so we're using `DataObject` instead of `ViewableData` here.